safesql/sqlwrap.go

```diff
@@ -0,0 +1,159 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package safesql
+
+import (
+	"context"
+	"database/sql"
+	"database/sql/driver"
+	"time"
+)
+
+func Drivers() []string { return sql.Drivers() }
+
+func Register(name string, driver driver.Driver) { sql.Register(name, driver) }
+
+type ColumnType = sql.ColumnType
+type DBStats = sql.DBStats
+type IsolationLevel = sql.IsolationLevel
+type NamedArg = sql.NamedArg
+type NullBool = sql.NullBool
+type NullFloat64 = sql.NullFloat64
+type NullInt32 = sql.NullInt32
+type NullInt64 = sql.NullInt64
```

Comment:
Is `NullUint32` and `NullUint64` not needed?

Reply:
doesn't seem to be in that package.